jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>Please move this up to the very top (for consistency and because it is the header that jupyter book uses for the table of contents that is rendered on the left etc).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>It's a sub-header of the section on how RNA velocity is modelled in practice. I feel like this section could be merged with the previous one. However, I don't think the section does not purely contain motivational content (first paragraph yes, the other no, IMO).</p><p>An alternative structure I thought of: Move the first paragraph of this section ("<em>The change in transcriptomic profile [...]</em>") to the <em>Motivation</em> section, and the remaining part to the <em>Parameter inference</em> section. The issue I see with this approach is that the section heading is talking about inferring parameters that have yet to be defined/talked about.</p><p>What do you think?</p>

Reply:
I think I got confused a little bit because I didn't see a top level header. I wasn't complaining about the order of the content and am happy with both options. Pick the one that is easier for you :)